src/System.Device.Gpio.Tests/README.md

```diff
@@ -0,0 +1,37 @@
+# How to run tests on the Raspberry Pi
+
+Prerequisites: 
+- Installed .NET SDK with the "dotnet" executable in the path
+- Locally cloned repository
+
+Build System.Device.Gpio.dll:
+```
+cd src/System.Device.Gpio
+dotnet build System.Device.Gpio.sln
+```
+This builds the main System.Device.Gpio assembly together with its test assembly. Before running the tests, you need to:
+- Connect BCM Pins 12 and 16 (physical Pins 32 and 36) with a 10kΩ resistor between them. 
+
+## Raspberry Pi driver tests
+After that, you can run the tests with the RaspberryPiDriver (which is the default low-level driver for the Raspberry Pi) like:
+```
+dotnet test --filter RaspberryPiDriverTests System.Device.Gpio.sln 
+```
+
+If everything went smoothly, the output should end with a success message. 
+
+## SysFsDriver Tests
+You can also run the SysFsDriver Tests (this uses a more generic approach). This driver requires root permissions to run, so you need to `sudo` the command:
+```
+sudo dotnet test --filter SysFsDriverTests System.Device.Gpio.sln 
```

Comment:
actually RPi driver also requires root, only libgpiod doesn't (at least that's what I was seeing)

Reply:
(I'll add some xunit traits in my PR to make it easier to run without sudo)

Reply:
That's weird. After a few retries, it _sometimes_ works without root rights and sometimes I get an exception.

Reply:
@krwq: It works without root rights each time except the very first time after bootup. If I run it twice (without sudo) it works, and after that it always works. Any idea?

Reply:
@pgrawehr this is good info. It might be a driver bug though but can't tell without investigating deeper

Reply:
I agree that this might just be a bug in our side. RPi driver shouldn't need sudo because it uses /dev/gpiomem/ which only exposes the gpio registers so it doesn't require sudo.

Reply:
I was also seeing locally it required sudo but I thought this was by design